### PR TITLE
Adding (x, y) hit to HGCal cell (u, v) for partial wafers v17  

### PR DESCRIPTION
**Pull request by Pruthvi-ch**
PR description:

There will no changes in the output.
No dependencies
This is related to getting cell id (u, v) form the (x, y) position inside a partial wafer.

PR validation:

Use runTheMatrix test workflows 'runTheMatrix.py -l limited -i all --ibeos'
Specific cfg's in Geometry/HGCalCommonData/test/python/testHGCalCellUVTester_cfg.py to test working

if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing

### CODE REVIEW

File: Geometry/HGCalCommonData/src/HGCalCellUV.cc [hunk 1]
```diff
@@ -266,3 +267,50 @@ std::pair<int, int> HGCalCellUV::cellUVFromXY4(double xloc,
...
   return uv;
 }
+
+std::pair<int32_t, int32_t> HGCalCellUV::cellUVFromXY1(
+    double xloc, double yloc, int32_t placement, int32_t type, int32_t partial, bool extend, bool debug) {
+  std::pair<int, int> uv = HGCalCellUV::cellUVFromXY1(xloc, yloc, placement, type, extend, debug);
+  int u = uv.first;
+  int v = uv.second;
+  if (partial == HGCalTypes::WaferLDTop) {
+    if (u > 7) {
```
Comment (cvuosalo) [inline Geometry/HGCalCommonData/src/HGCalCellUV.cc:277]: 6, 7, and 10 look like magic numbers in this code. They should probably be replaced by constants.

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by cvuosalo** [comment]
+1

**Comment by bsunanda** [comment]
@srimanob Could you please approve this PR. Needed to define V17 geometry

**Comment by bsunanda** [comment]
@srimanob Could you please approve this PR. Needed to define V17 geometry

**Comment by srimanob** [comment]
+Upgrade

This PR is adding function to get (u,v) from (x,y) hit, used for HGCal V17.

**Comment by perrotta** [comment]
+1

**Pull request merged**